package-lock.json

```diff
@@ -0,0 +1,11609 @@
+{
```

Comment:
this is more like a question for understanding :  If we commit this file, do we have to do "npm install" each time after pulling new commits in general ? wouldnt this change this file again, triggering new changes and so on ?

Reply:
The file now shows the exact versioning (including the explicit hashes). Thus, an `npm i` is carried out more quickly if we version this file as well. In general, in my world, it is advisable to perform an `npm i` when creating a new branch anyway.

Reply:
ok, thanks for the information. Will do this when i create a feature branch.